data/human/free worlds epilogue.txt

```diff
@@ -197,3 +197,96 @@ mission "FW Epilogue: Danforth"
+			choice
+				`	"Alright, I suppose for old times' sake I can help out."`
+				`	"Sorry, I just wanted to check in and see how you were doing."`
+					goto catchup
+			`	He turns to you and smirks. "We've recommissioned the ship from retirement, so it's not top of the line, but try to keep it in one piece for future use. Understand?"`
+			choice
+				`	"Of course! I'll take care of it."`
+			`	"Great," he says, rubbing his hands together with a gleeful expression on his face. He's clearly looking forward to hearing what you think of his latest invention.`
+				accept
+			label catchup
+			`	"I see. Well, I'm sure we can find someone else up to the task. Regardless, Captain <last>, I assure you this new weapon vastly improves on the outdated plasma cannon. This'll put Kraz Cybernetics in the big leagues!"`
+			`	As his ramblings about his new invention continue, you think to yourself about the old times, the war, and all the people you've met along the way.`
+			`	"How about we get out of here, <first>? My work is done for today, and I know a homely little bar that's only a short walk."`
+			choice
+				`	"Of course. It's not like there's a war to worry about anymore."`
+			`"Ha! Well, of course, the war kept all of us plenty busy." He leads you out of the facility while telling an anecdote that must be humor for an engineer.`
+			`	You follow Barmy into the bar. Even with the smear of grease still on his face, he fits in with the crowd. You both stand at the bar and order your drinks. Barmy's order catches you off guard, and he notices your expression. "What? A man who works with his hands can't enjoy a drink with some age?" You chuckle, and you both tell stories of your early lives. He tells how he had some exciting adventures in his youth among the stars but found his passion tinkering and finding ways to improve on outdated equipment.`
+			`	He's shocked to discover your origins on New Boston. "You're really from that backwater planet? I'm surprised you were able to escape the racket they have going on there. I was lucky to get out myself!"`
+			choice
+				`	"It wasn't perfect, but at least it wasn't quite so noisy down on the ground."`
+				`	"Nothing like it is now. I've got the whole universe ahead of me!"`
+			`	"You wouldn't think to have met someone from your very own birth planet, eh?" He chuckles. "Small galaxy, I suppose." You both have a great time hearing of each other's early lives, but you eventually wrap up your visit after a long night of bouncing stories back and forth. You say your goodbyes and head back to your ship.`
+				decline
+		event "plasma repeater advanced" 95
+		event "plasma repeater basics" 155
```

Comment:
I believe these should be in an on decline.
```suggestion
	on decline
		event "plasma repeater advanced" 95
		event "plasma repeater basics" 155
```

Reply:
But since the `on complete` section also has `decline` endpoints, would that not run it twice or necessitate using the same event timer for both branches?

Reply:
`on decline` only triggers when the mission itself is declined, not when any conversation related to the mission hits a decline endpoint.